swagger/pelican-swagger.yaml

```diff
@@ -418,6 +418,20 @@ definitions:
   OriginExport:
     type: object
     properties:
+      status:
+        type: string
+        description:
+          The registration status of the federation prefix.
+          Can be "Not Supported"|"Completed"|"Incomplete"|"Registration Error"
```

Comment:
How is "Not Supported" different from "Registration Error"?

Reply:
"Not Support" is when the registry is below 7.8.0 (or whichever version this PR is included in)
"Registration Error" means that there's an issue with the registration itself, like not having identity when registering a namespace overlapped with topology, or something else that leads to a failure of registering the namespace. In this case, the "completeness status" doesn't make any sense because the namespace is not registered at all.

Reply:
I will update the Swagger doc to detail them